cmd/alter/alter-topic.go

```diff
@@ -0,0 +1,30 @@
+package alter
+
+import (
+	"github.com/deviceinsight/kafkactl/cmd/validation"
+	"github.com/deviceinsight/kafkactl/operations"
+	"github.com/spf13/cobra"
+)
+
+var flags operations.CreateTopicFlags
+
+var cmdAlterTopic = &cobra.Command{
+	Use:   "topic",
+	Short: "alter a topic",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		(&operations.TopicOperation{}).AlterTopic(args[0], flags)
+	},
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return validation.ValidateAtLeastOneRequiredFlag(cmd)
+	},
+}
+
+func init() {
+	cmdAlterTopic.Flags().Int32VarP(&flags.Partitions, "partitions", "p", flags.Partitions, "number of partitions")
+	cmdAlterTopic.Flags().StringArrayVarP(&flags.Configs, "config", "C", flags.Configs, "configs in format `key=value`")
```

Comment:
change shorthand to lowercase "c"

Reply:
Done. The current master actually still has an issue with the global `-C` shorthand. I added a fix as well.